packages/mobile/src/redux/migrations.ts

```diff
@@ -416,4 +416,11 @@ export const migrations = {
       hasLinkedBankAccount: false,
     },
   }),
+  33: (state: any) => ({
+    ...state,
+    app: {
+      ...state.app,
+      ranVerificationMigrationAt: null,
```

Comment:
This is the actual relevant change. By setting the field as `null` the verification migration will be run again